src/metabase/api/common.clj

```diff
@@ -419,6 +419,17 @@
     (check (not (:archived object))
       [404 {:message (tru "The object has been archived."), :error_code "archived"}])))
 
+(defn check-valid-limit
+  "Check on paginated stuff that, if the offset exists, the limit exists."
+  [limit offset]
+  (check (not (and offset (not limit))) [400 (tru "When including an offset, a limit must also be included.")]))
+
+(defn check-valid-offset
```

Comment:
Does it make sense to combine this with `check-valid-limit` and just perform a single check for both error conditions, since the param lists are identical?  Then only one would need invoked below too.

Reply:
I didn't want any i18n changes (cuz lol) but I can just do two checks in the one check, I guess